funccount.py

```diff
@@ -0,0 +1,30 @@
+import os
+import signal
+import subprocess as sp
+import time
+
+class FUNCCOUNT:
+    def __init__(self, funcs, sleep=0):
+        self.funcs = funcs
+        self.sleep = sleep
+
+    def start(self):
+        cmd = f"sleep {self.sleep} && funccount {self.funcs}"
```

Comment:
If you're going to use shell=True, then you need to be careful about quoting your arguments, as you would with a shell script....

But I try to avoid using the shell inside a python script if at all possible. A thread with a delay on a stop Event before calling funccount would probably be a bit nicer.